releases/release-5.2.4.md

```diff
@@ -0,0 +1,259 @@
+    - (dup: release-5.3.1.md > Bug fixes> TiKV)- Fix the metadata corruption issue when `Prepare Merge` is triggered after a new election is finished but the isolated peer is not informed [#11526](https://github.com/tikv/tikv/issues/11526)
+    - (dup: release-5.3.1.md > Bug fixes> TiKV)- Fix the issue that the latency of Resolved TS increases after TiKV stops operating [#11351](https://github.com/tikv/tikv/issues/11351)
+    - (dup: release-5.3.1.md > Bug fixes> TiKV)- Fix a panic issue that occurs when Region merge, ConfChange, and Snapshot happen at the same time in extreme conditions [#11475](https://github.com/tikv/tikv/issues/11475)
+    - (dup: release-5.3.1.md > Bug fixes> TiKV)- Fix a bug that tikv-ctl cannot return the correct Region-related information [#11393](https://github.com/tikv/tikv/issues/11393)
+    - (dup: release-5.0.6.md > Bug fixes> TiKV)- Fix the issue of negative sign when the decimal divide result is zero [#29586](https://github.com/pingcap/tidb/issues/29586)
+    - (dup: release-5.4.0.md > Bug fixes> TiKV)- Fix the issue that retrying prewrite requests in the pessimistic transaction mode might cause the risk of data inconsistency in rare cases [#11187](https://github.com/tikv/tikv/issues/11187)
+    - (dup: release-5.3.0.md > Bug Fixes> TiKV)- Fix a memory leak caused by monitoring data of statistics threads [#11195](https://github.com/tikv/tikv/issues/11195)
+
++ PD
+
+    - (dup: release-6.0.0-dmr.md > Bug fixes> PD)- Fix the issue that the Region scatterer scheduling lost some peers [#4565](https://github.com/tikv/pd/issues/4565)
+    - (dup: release-5.4.0.md > Bug fixes> PD)- Fix the issue that the cold hotspot data cannot be deleted from the hotspot statistics [#4390](https://github.com/tikv/pd/issues/4390)
+
++ TiDB Dashboard
+
+    -
+
++ TiFlash
+
+    - (dup: release-6.0.0-dmr.md > Bug fixes> TiFlash)- Fix a bug that MPP tasks might leak threads forever [#4238](https://github.com/pingcap/tiflash/issues/4238)
+    - (dup: release-6.0.0-dmr.md > Bug fixes> TiFlash)- Fix the issue that the result of `IN` is incorrect in multi-value expressions [#4016](https://github.com/pingcap/tiflash/issues/4016)
+    - (dup: release-6.0.0-dmr.md > Bug fixes> TiFlash)- Fix the issue that casting `INI` to `DECIMAL` might cause overflow [#3920](https://github.com/pingcap/tiflash/issues/3920)
+    - (dup: release-6.0.0-dmr.md > Bug fixes> TiFlash)- Fix the issue that the date format identifies `'\n'` as an invalid separator [#4036](https://github.com/pingcap/tiflash/issues/4036)
+    - (dup: release-6.0.0-dmr.md > Bug fixes> TiFlash)- Fix the potential query error after adding columns under heavy read workload [#3967](https://github.com/pingcap/tiflash/issues/3967)
+    - (dup: release-6.0.0-dmr.md > Bug fixes> TiFlash)- Fix the wrong result that occurs when casting `DATETIME` to `DECIMAL` [#4151](https://github.com/pingcap/tiflash/issues/4151)
```

Comment:
```suggestion
+ TiFlash

    - Fix the bug that MPP tasks might leak threads forever [#4238](https://github.com/pingcap/tiflash/issues/4238)
    - Fix the issue that the result of `IN` is incorrect in multi-value expressions [#4016](https://github.com/pingcap/tiflash/issues/4016)
    - Fix the issue that the date format identifies `'\n'` as an invalid separator [#4036](https://github.com/pingcap/tiflash/issues/4036)
    - Fix the potential query error after adding columns under heavy read workload [#3967](https://github.com/pingcap/tiflash/issues/3967)
    - Fix the bug that invalid storage dir configurations lead to unexpected behavior [#4093](https://github.com/pingcap/tiflash/issues/4093)
    - Fix the bug that some exception is not handle is not handled properly [#4101](https://github.com/pingcap/tiflash/issues/4101)
    - Fix the bug that function `STR_TO_DATE()` incorrectly handles leading zeros when parsing microseconds [#3557](https://github.com/pingcap/tiflash/issues/3557)
    - Fix the issue that casting `INI` to `DECIMAL` might cause overflow [#3920](https://github.com/pingcap/tiflash/issues/3920)
    - Fix the wrong result that occurs when casting `DATETIME` to `DECIMAL` [#4151](https://github.com/pingcap/tiflash/issues/4151)
    - Fix the wrong result that occurs when casting `FLOAT` to `DECIMAL` when overflow happens [#3998](https://github.com/pingcap/tiflash/issues/3998)
    - Fix the issue that `castStringAsReal` behavior is inconsistent in TiFlash and in TiDB/TiKV [#3475](https://github.com/pingcap/tiflash/issues/3475)
    - Fix the issue that `CastStringAsDecimal` behavior is inconsistent in TiFlash and in TiDB/TiKV [#3619](https://github.com/pingcap/tiflash/issues/3619)
    - Fix the issue that TiFlash might return the `EstablishMPPConnection` error after it is restarted [#3615](https://github.com/pingcap/tiflash/issues/3615)
    - Fix the issue that obsolete data cannot be reclaimed after set tiflash replica to 0 [#3659](https://github.com/pingcap/tiflash/issues/3659)
    - Fix the potential data inconsistency caused by widening the primary key column when the primary key is handle [#3569](https://github.com/pingcap/tiflash/issues/3569)
    - Fix the possible parsing errors when an SQL statement contains extremely long nested expressions [#3354](https://github.com/pingcap/tiflash/issues/3354)
    - Fix the possible wrong result that queries with the `where <string>` clause [#3447](https://github.com/pingcap/tiflash/issues/3447)
    - Fix the possible wrong result when new collation is enabled [#3388](https://github.com/pingcap/tiflash/issues/3388), [#3391](https://github.com/pingcap/tiflash/issues/3391)
    - Fix the panic issue that occurs when TLS is enabled [#4196](https://github.com/pingcap/tiflash/issues/4196)
    - Fix the panic issue that occurs when the memory limit is enabled [#3902](https://github.com/pingcap/tiflash/issues/3902)
    - Fix the panic issue that occurs when an MPP query is stopped [#3401](https://github.com/pingcap/tiflash/issues/3401)
    - Fix the unexpected error of `Unexpected type of column: Nullable(Nothing)` [#3351](https://github.com/pingcap/tiflash/issues/3351)
```

Reply:
`- Fix the problem that empty segments cannot be merged after gc [#4511](https://github.com/pingcap/tiflash/issues/4511)` is removed since it is not merged into release-5.2 branch

Reply:
Found a typo in the 6.0.0 release note https://github.com/pingcap/docs/pull/8306